src/Lint.jl

```diff
@@ -335,7 +341,7 @@ function lintinclude(ctx::LintContext, file::AbstractString)
 end
 
 """
-Lint all *.jl files at a given directory.
+Lint all \*.jl files at a given directory.
```

Comment:
I guess you're escaping the markdown here, but I don't think this works. Need two backslashes, that is `\\*`, because `\*` is the same as `*` in a string.